### PR TITLE
Add ratatool-scalacheck generators for TableRow and ProtoBuf

### PR DESCRIPTION
**Pull request by idreeskhan**
Addresses #27, uses Scalacheck `Gen[T]` Monad to create generators.

### CODE REVIEW

File: ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/ProtoBufGenerator.scala [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      case Type.UINT32 => genPositiveInt.map(v => UInt32Writer(id, v))
+      case Type.UINT64 => genPositiveLong.map(v => UInt64Writer(id, v))
+      case Type.SINT32 => genPositiveInt.map(v => SInt32Writer(id, v))
+      case Type.SINT64 => genPositiveLong.map(v => SInt64Writer(id, v))
+      case Type.FIXED32 => genPositiveInt.map(v => Fixed32Writer(id, v))
+      case Type.FIXED64 => genPositiveLong.map(v => Fixed64Writer(id, v))
+      case Type.SFIXED32 => genPositiveInt.map(v => SFixed32Writer(id, v))
+      case Type.SFIXED64 => genPositiveLong.map(v => SFixed64Writer(id, v))
+      case Type.BOOL => Arbitrary.arbBool.arbitrary.map(v => BoolWriter(id, v))
+      case Type.STRING => Gen.choose(0, 40)
```
Comment (jbigred1) [inline ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/ProtoBufGenerator.scala]: what's this magic range for

should we add a comment to explain

**Comment by jbigred1** [comment]
looks good 👍

**Comment by ankscircle** [comment]
:shipit:

File: ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/ProtoBufGenerator.scala (hunk 1 above)
Comment (idreeskhan) [reply ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/ProtoBufGenerator.scala]: Mostly to mimic behaviour here https://github.com/spotify/ratatool/blob/master/ratatool/src/main/scala/com/spotify/ratatool/generators/ProtoBufGenerator.scala#L85

I'm not certain why this is limited to begin with though. @nevillelyh do you know?

Comment (idreeskhan) [reply ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/ProtoBufGenerator.scala]: I'll move to an arbitrary string for now, can revert later if needed

**Pull request merged**